commit c4e769e5436f7d5f0d3a296e40a9e2bcae791ef7 (refs/changes/60/4560/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-07-24T10:33:00-07:00 (1 year, 2 months ago)
    
    MANTA-3849 'manta-adm update' takes a '--config-file' opt but does not use it

diff --git a/cmd/manta-adm.js b/cmd/manta-adm.js
index f277622..8a74777 100755
--- a/cmd/manta-adm.js
+++ b/cmd/manta-adm.js
@@ -538,13 +538,15 @@ MantaAdm.prototype.do_update = function (subcmd, opts, args, callback)
 };
 
 MantaAdm.prototype.do_update.help =
-    'Update deployment to match a JSON configuration.\n\n{{options}}';
+    'Update deployment to match a JSON configuration.\n\n' +
+    'Usage:\n\n' +
+    '    manta-adm update [OPTIONS] CONFIG-FILE [SERVICE]\n\n' +
+    '{{options}}';
 
 MantaAdm.prototype.do_update.options = [
     maCommonOptions.logFile,
     maCommonOptions.dryrun,
     maCommonOptions.confirm,
-    maCommonOptions.configFile,
 {
     'names': [ 'no-reprovision' ],
     'type': 'bool',
